db/view/view_update_backlog.hh

```diff
@@ -50,11 +50,11 @@ public:
         return float(_current) / float(_max) / admission_control_threshold;
     }
 
-    const size_t& get_current_bytes() const {
+    const size_t& get_current_size() const {
```

Comment:
This change deserves adding a comment - if "size" is not "bytes", what is it? What if the write queue has 100,000 bytes and the read queue has 17 reads, what "size" is returned?